scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 511,
```

Comment:
Does this really only increase the max? does the min not need to go up too?

Reply:
You're right looks like the smallest bundle is now 514kb, I'll amend it 
<img width="267" alt="Screenshot 2019-08-27 at 13 34 56" src="https://user-images.githubusercontent.com/4706195/63771679-8401b280-c8cf-11e9-9bd3-d7f9cb14a3da.png">